# which go to the model you want to use
cfg_root: "/home/kin/workspace/OpenPCDet/tools/cfgs/kitti_models/pv_rcnn.yaml"

# which go to the .pth
model_path: "/home/kin/workspace/OpenPCDet/tools/pv_rcnn_8369.pth"

# only the predict probablity over this threshold will be draw a box
threshold: 0.8

move_lidar_center: 20
viz_rate: 1.0